src/Sarif.Converters/PylintConverter.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.CodeAnalysis.Sarif.Writers;
+using Microsoft.CodeAnalysis.Sarif.Converters.PylintObjectModel;
+
+namespace Microsoft.CodeAnalysis.Sarif.Converters
+{
+    public class PylintConverter : ToolFileConverterBase
+    {
+        public override void Convert(Stream input, IResultLogWriter output, LoggingOptions loggingOptions)
+        {
+            input = input ?? throw new ArgumentNullException(nameof(input));
+            output = output ?? throw new ArgumentNullException(nameof(output));
+
+            PylintLog PylintDefects = PylintLogReader.ReadLog(input);
```

Comment:
Local variable names are camelCase, so `pyLintDefects`. But you're creating a log, so `log` is a perfectly good name for this variable.